logstash_async/transport.py

```diff
@@ -43,6 +43,8 @@ class Transport(ABC):
     :type host: str
     :param port: The TCP/UDP port.
     :type port: int
+    :param path: The path for the transport.
```

Comment:
I think we don't need the `path` argument in the base class. The `path` is probably quite specific for HTTP and so IMO not suitable for the base class.